terraform/linode.tf

```
@@ -0,0 +1,52 @@
+#
+# Nimbus
+# Linode Terraform Deployment
+#
+
+locals {
+  domain      = "mrzzy.co"
+  domain_slug = replace(local.domain, ".", "-")
+
+  # Linode deploy region
+  linode_region = "ap-south" # singapore
+}
+
+# Linode Cloud
+provider "linode" {}
+
+# Linode: LKE Cluster
+module "k8s" {
+  source = "./modules/linode/k8s"
+  region = local.linode_region
+
+  machine_type = "g6-standard-2" # 2vCPU, 2GB
+  n_workers    = 1
+
+  # TLS credentials to add to the cluster as K8s secrets.
+  tls_certs = {
+    "${local.domain_slug}-tls" = {
+      "cert" = module.tls_cert.full_chain_cert,
+    }
+  }
+  tls_keys = {
+    "${local.domain_slug}-tls" = module.tls_cert.private_key,
+  }
+}
+
+# Linode: DNS zone & routes for mrzzy.co domain
+locals {
+  warp_ip = module.warp_vm.external_ip
+}
+module "dns" {
+  source = "./modules/linode/dns"
+
+  domain = "mrzzy.co"
```

```suggestion
  domain = local.domain
```
Already defined local for this.